java-frontend/src/test/files/se/ConditionAlwaysTrueOrFalseCheck.java

```diff
@@ -2123,3 +2123,14 @@ void bar(A a) {
     }
   }
 }
+
+class OptionalEmptyNotPresent {
```

Comment:
I would also add a test on the `OptionalGetBeforeIsPresentCheck` test file to show that we are raising issue explicitly on them as well.